components/dashboard/src/components/ContextMenu.tsx

```diff
@@ -106,7 +106,7 @@ function ContextMenu(props: ContextMenuProps) {
             </div>
             {expanded ? (
                 <div
-                    className={`mt-2 z-50 bg-white dark:bg-gray-900 absolute flex flex-col border border-gray-200 dark:border-gray-800 rounded-lg truncated ${
+                    className={`mt-2 z-50 bg-white dark:bg-gray-900 absolute flex flex-col border border-gray-200 dark:border-gray-800 drop-shadow-xl rounded-lg truncated ${
```

Comment:
In my testing, I seemed to need to also add the `filter` class for our version of tailwind in order for the dropshadow to show up.